pkg/apply/solver/solver_test.go

```diff
@@ -378,7 +384,7 @@ func TestTaskQueueBuilder_AppendApplyWaitTasks(t *testing.T) {
 					actWaitTask := toWaitTask(t, actualTask)
 					assert.Equal(t, len(expTsk.Ids), len(actWaitTask.Ids))
 					// Order is NOT important for ids stored within task.
-					if !object.SetEquals(expTsk.Ids, actWaitTask.Ids) {
+					if !object.IsSetEqual(expTsk.Ids, actWaitTask.Ids) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.